### PR TITLE
Docs css and link updates

### PR DESCRIPTION
**Pull request by conceptualshark**
### Code Changes

* Prepped last of the docs for 2.0 release 
* Removed lingering fidesctl/fidesops references
* Update footer links to 2.0 docs
* Updated CSS to appropriate brand coloring
* 
### Steps to Confirm

* build the docs successfully

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [x] Documentation Updated
* [x] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: mkdocs/docs/syntax.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The Fides language is an intentionally simple language designed to be relatively
 
 ### Fides Taxonomy
 
-The Fides language is intentionally simple. To assure this, Fides declarations use predefined primitives (e.g. data categories) that are used when describing your datasets, systems, policies, etc. These predefined primitives exist as part of the Fides taxonomy which is maintained in your `fidesctl` server so they can be consistently used across your organization's development team. 
+The Fides language is intentionally simple. To assure this, Fides declarations use predefined primitives (e.g. data categories) that are used when describing your datasets, systems, policies, etc. These predefined primitives exist as part of the Fides taxonomy which is maintained in your Fides server so they can be consistently used across your organization's development team.
```
Comment (ThomasLaPiana) [inline mkdocs/docs/syntax.md:12]: I always cringe at the word `simple`, it is subjective and can make people feel unintelligent/unworthy of using the product if they find it difficult.

Maybe something like "we attempt to keep the Fides language as straightforward as possible" or something of that nature? What do you think?

Comment (ThomasLaPiana) [reply mkdocs/docs/syntax.md:12]:
```suggestion
The Fides language is intentionally simple. To ensure this, Fides declarations use predefined primitives (e.g. data categories) that are used when describing your datasets, systems, policies, etc. These predefined primitives exist as part of the Fides taxonomy which is maintained in your Fides server so they can be consistently used across your organization's development team. 
```

[I think this is technically correct?](https://newsroom.unl.edu/announce/snr/4471/25500)

File: mkdocs/docs/taxonomy/data_categories.md [hunk 2]
```diff
@@ -68,10 +68,10 @@ Below is a reference for all subcategories of `system` and `user` to assist with
 | `browsing_history`                | `user`                                    |Content browsing history of a user.                                                            |
 | `contact`                         | `user`                                    |User contact data.                                                                             |
 | `address`                         | `user.contact`                            |User contact data related to an address.                                                       |
+| `email`                           | `user.contact`                    |User's email address.                                                                          |
```
Comment (ThomasLaPiana) [inline mkdocs/docs/taxonomy/data_categories.md]: formatting got bumped 🤔

**Review by ThomasLaPiana — CHANGES_REQUESTED** [review]
Some small comments, overall awesome update!

**Review by NevilleS — APPROVED** [review]
Incorporated @ThomasLaPiana's feedback and merging, thanks

**Pull request merged**